docs/standard/serialization/preserve-references.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
File name -- for consistency with the others, should be prefixed by system-text-json-, i.e., system-text-json-preserve-references.md